doc/source/user_information/how_tos/agenda.rst

```diff
@@ -489,6 +489,60 @@ Note that the ``config`` section still applies to every spec in the agenda. So
 the precedence order is -- spec settings override section settings, which in
 turn override global settings.
 
+
+.. _section-groups:
+
+Section Groups
+---------------
+
+Section groups are a way of grouping sections together and are used to produce a
+cross product of each of the different groups. This can be useful when you want
+to run a set of experiments with all the available combinations without having
+to specify each combination manually.
+
+For example if we want to investigate the differences between running the
+maximum and minimum frequency with both the maximum and minimum number of cpus
+online, we can create an agenda as follows:
+
+.. code-block:: yaml
+
+        sections:
+          - id: min_freq
+          runtime_parameters:
+              freq: min
+          group: frequency
+         - id: max_freq
+          runtime_parameters:
+              freq: max
+         group: frequency
+
+         - id: min_cpus
+           runtime_parameters:
+              cpus: 1
+         group: cpus
+         - id: max_cpus
+           runtime_parameters:
+              cpus: 8
+         group: cpus
+
+        workloads:
+        -  dhrystone
+
+This will results in 8 jobs being generated for each of the possible
+combinations with the relevant :ref:`classifiers <classifiers>` automatically
```

Comment:
Should describe what the "relevant" classifiers are.

Reply:
Updated.